File: python/CustomException.py
class InvalidNumberException(Exception):
    def __init__(self):
        super().__init__("Angka yang dimasukkan melanggar ketentuan")


def cekAngka(angka):
    if angka < 0 or angka > 20:
        return True
    else:
        raise InvalidNumberException()


try:
    cekAngka(20)
except Exception as e:
    print("Terjadi error:\n", str(e))

File: java/CustomException.java
class InvalidNumberException extends Exception {
    public InvalidNumberException() {
        super("Angka yang dimasukkan melanggar ketentuan");
    }
}

public class CustomException {
    public static boolean cekAngka(int angka) throws InvalidNumberException {
        if (angka < 0 || angka > 20)
            return true;
        else
            throw new InvalidNumberException();
    }

    public static void main(String[] args) {
        try {
            cekAngka(20);
        } catch (Exception e) {
            System.out.println("Terjadi error : \n" + e.getMessage());
        }
    }
}